e3db/tests/test_tsv1_auth.py

```diff
@@ -3,8 +3,8 @@
 
 # Credentials
 CREDS = {
-	"private_signing_key": "d55u6bLR9tkMVA4OwYIPepOOeXVSHHEit8VoXGRMQiaf5wKRk9gooP9pN3LBJ28BIW9fZ9-ZZPLVsHtuPqkRSQ",
-	"public_signing_key": "n-cCkZPYKKD_aTdywSdvASFvX2ffmWTy1bB7bj6pEUk",
+    "private_signing_key": "d55u6bLR9tkMVA4OwYIPepOOeXVSHHEit8VoXGRMQiaf5wKRk9gooP9pN3LBJ28BIW9fZ9-ZZPLVsHtuPqkRSQ",
+    "public_signing_key": "n-cCkZPYKKD_aTdywSdvASFvX2ffmWTy1bB7bj6pEUk",
```

Comment:
intentional?

Reply:
Oops, ya that was an accident. They were temporarily removed while testing other signing keys when I was having issues with generating matching signatures between SDKs. I put them back in so that the `KNOWN_SIGNATURE` and `KNOWN_HASH` values in the tests would still work. Will bump those back inline with the fields.